content/hi/service.md

```diff
@@ -0,0 +1,12 @@
+---
+title: सेवा
```

Comment:
please add the english name of term in parenthesis:
e.g title: सर्वरलेस (Serverless)